src/scripts/ugg.js

```diff
@@ -0,0 +1,88 @@
+/*
+*	U.GG
+*/
+chrome.runtime.sendMessage({
+    from:    'content',
+    subject: 'showPageAction'
+});
+
+chrome.runtime.onMessage.addListener(function(msg, sender, response) {
+    if ((msg.from === 'popup') && (msg.subject === 'guide')) {
+
+		var BLOCKS = {
+			title: null, 
+			items: []
+		};
+		var title = stripURL($('.champion-image').attr('src')) + " - U.GG";
+		BLOCKS.title = title;
+
+		for (i = 0; i < 2; i++) {
+			var fields = ['.starting-items','.final-items'];
+			$(fields[i] + ' .items').each(function () {
+				var header = getTitle(i);
+				var items = [];
+
+				$(this).children('.image-wrapper').each(function () {
```

Comment:
I think we can create a helper function here to work with the different scenarios